source/howto/select-field-type.md

```diff
@@ -160,5 +160,26 @@ comma. The following example will display the 'first name' and 'last name' of ea
 ```
 
 
+If the list is growing longer, there are a few ways to make it more manageable:
 
+```apache
```

Comment:
highlight.js doesn't support `yml`, and `apache` looks nicer than whatever it picks as a fallback. ;-)